worker/src/handles/TcpConnectionHandle.cpp

```diff
@@ -232,6 +235,29 @@ void TcpConnectionHandle::Write(
 		return;
 	}
 
+#ifdef MS_LIBURING_SUPPORTED
+	{
+		if (!DepLibUring::IsActive())
+		{
+			goto write_libuv;
+		}
+
+		// Prepare the data to be sent.
+		// NOTE: If all SQEs are currently in use or no UserData entry is available we'll
+		// fall back to libuv.
+		auto prepared = DepLibUring::PrepareWrite(this->fd, data1, len1, data2, len2, cb);
+
+		if (!prepared)
+		{
+			// Cannot write via liburing, fallback to libuv.
+			goto write_libuv;
+		}
+
+		return;
+	}
```

Comment:
We don't need those brackets.

```suggestion
	if (!DepLibUring::IsActive())
	{
		goto write_libuv;
	}

	// Prepare the data to be sent.
	// NOTE: If all SQEs are currently in use or no UserData entry is available we'll
	// fall back to libuv.
	auto prepared = DepLibUring::PrepareWrite(this->fd, data1, len1, data2, len2, cb);

	if (!prepared)
	{
		// Cannot write via liburing, fallback to libuv.
		goto write_libuv;
	}

	return;
```

Reply:
The brackets are there for a reason. Try to remove them and it won't compile:

<img width="1389" alt="Screenshot 2023-11-20 at 16 58 35" src="https://github.com/versatica/mediasoup/assets/732340/1968823e-ece1-4003-98ba-b83aed63f4c3">

Reply:
Hell, ok.